packages/bitcoin/src/bitcoin-signer.ts

```diff
@@ -1,13 +1,40 @@
-import { HDKey } from '@scure/bip32';
+import { HARDENED_OFFSET, HDKey } from '@scure/bip32';
```

Comment:
Will we split this up into a sub package / smaller files?

Reply:
We can create sub-esm packages like `@leather.io/bitcoin/signers`, but would only bother with that when bloat becomes an issue